obs2ioda-v2/src/Makefile

```diff
@@ -14,7 +14,8 @@
 FC = ifort
 #BUFR_LIB = -L/glade/u/home/hclin/extlib/intel -lbufr
 #FFLAGS = -mcmodel medium # needed for intel error message "failed to convert GOTPCREL relocation"
-BUFR_LIB = -L/glade/campaign/mmm/parc/ivette/pandac/converters/WRFDA_3DVAR_dmpar/var/external/bufr -lbufr
+
+BUFR_LIB = -L/glade/u/home/mrislam/WRFDA/WRFDA-3DVAR/var/external/bufr -lbufr #-L/glade/p/mmm/parc/ivette/pandac/converters/WRFDA_3DVAR_dmpar/var/external/bufr -lbufr #-L/glade/u/home/hclin/extlib/intel -lbufr
```

Comment:
Do you have to change the bufr lib location for compilation? though this may not be relevant anymore as @amstokely 's PR will switch to NCEP bufr lib.